src/server/plugins/engine/models/SummaryViewModel.ts

```diff
@@ -187,71 +169,70 @@ export class SummaryViewModel {
 
     return details
   }
-
-  private getRelevantPages(model: FormModel, state: FormSubmissionState) {
```

Comment:
I've consolidated the "page walkers" in here and in `getConditionEvaluationContext()`

We now have an all-in-one `model.getFormContext()` which includes:

```js
{
  evaluationState: FormState
  relevantState: FormState
  relevantPages: PageControllerClass[]
  paths: string[]
}
```